Makefile

```diff
@@ -62,6 +65,13 @@ $(THANOS): $(BIN_DIR)
 	tar xvfz ./tmp/thanos.tar.gz -C ./tmp
 	mv ./tmp/thanos-0.11.0.linux-amd64/thanos $@
 
+$(LOKI): $(BIN_DIR)
+	(loki_pkg="loki-$$(go env GOOS)-$$(go env GOARCH)" && \
```

Comment:
why is all of this wrapped in  a `()`? Is it to put it in the same subshell? It should be enough that everything is `&&`ed and joined with `\`. I don't think we need to put this in a subshell